fixtures/util.go

```diff
@@ -0,0 +1,173 @@
+package fixtures
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/go-git/go-git/v5"
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+	"gopkg.in/yaml.v2"
+)
+
+// CloneRepo method to clone dev-analytics-api repo as it contains the fixtures files
+func CloneRepo() error {
+	token := os.Getenv("GITHUB_OAUTH_TOKEN")
+	stage := os.Getenv("STAGE")
+	repo := "github.com/LF-Engineering/dev-analytics-api.git"
+	url := fmt.Sprintf("https://%s@%s", token, repo)
+	options := git.CloneOptions{
+		URL:      url,
+		Progress: os.Stdout,
+	}
+	_, err := git.PlainClone("./data", false, &options)
+	if err != nil {
+		log.Printf("Error cloning repo: %+v", err)
+	}
+
+	// checkout branch will be test or prod based on STAGE env var
+	cmd := exec.Command("git", "checkout", stage)
+	cmd.Dir = "./data"
+	err = cmd.Run()
+	if err != nil {
+		return err
+	}
+
+	// if repo cloned already pull latest
+	cmd = exec.Command("git", "pull", "origin", stage)
+	cmd.Dir = "./data"
+	err = cmd.Run()
+	if err != nil {
+		return err
+	}
+
+	return err
+}
+
+// GetYamlFiles returns all the fixture yaml files
+func GetYamlFiles() []string {
+	var result = make([]string, 0)
+	err := filepath.Walk("./data/app/services/lf/bootstrap/fixtures",
+		func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+			if strings.Contains(path, "yaml") {
```

Comment:
lets use `filepath.Ext(path)`. that way we only get the `.yaml`  extension instead of any occurance of `yaml` in the string.

Reply:
done